lib/datasource/cdnjs/common.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from 'zod';
+import type { CdnjsResponse } from './types';
+
+/**
+ * See: https://cdnjs.com/api
+ */
+export const Response = z.object({
+  homepage: z.string().url().optional(),
+  repository: z
+    .object({
+      url: z.string().url().optional(),
+    })
+    .optional()
+    .nullable(),
+  assets: z.array(
+    z.object({
+      version: z.string(),
+      files: z.array(z.string()),
+      sri: z.record(z.string()),
+    })
+  ),
+});
+
+// export type IResponse = z.infer<typeof Response>;
```

Comment:
All these libraries have similar type inference feature (library-based DSL -> typescript types), but it won't work correctly without strict nulls. You could uncomment this line and inspect resulting type in VSCode to see what I mean. Then you can temporarily enable strict nulls and see the difference.

Since this files is included in `tsconfig.strict.ts`, we always check that library-based definition conforms older TypeScript-based one. Once strict nulls are enabled globally, we're just switching and removing definitions from `types.ts`.